src/dotnet-test-nunit/project.json

```diff
@@ -34,7 +34,9 @@
         "Microsoft.NETCore.App": {
           "version": "1.0.0",
           "type": "platform"
-        }
+        },
+        "System.Xml.XmlDocument": "4.0.1",
```

Comment:
I didn't realize this was available now. I thought we had to wait for .NET Standard 2.0. This will make our Engine code changes we need to do easier 👍